src/Agent.php

```diff
@@ -287,21 +296,30 @@ public function showForm($id, array $options = [])
             echo "<tr class='tab_bg_1'>";
             echo "<td>" . __('Network inventory threads') . "</td>";
             echo "<td>";
+            $general = __('General setup');
+            if (isset($CFG_GLPI['threads_networkinventory'])) {
+                $general = sprintf('%1$s (%2$s)', $general, $CFG_GLPI['threads_networkinventory']);
+            }
             Dropdown::showNumber(
                 'threads_networkinventory',
                 [
-                'value' => $this->fields['threads_networkinventory'],
-                'toadd' => [__('General setup')]
+                    'value' => $this->fields['threads_networkinventory'],
+                    'toadd' => [$general],
+                    'min' => 1
                 ]
             );
             echo "</td>";
             echo "<td>" . __('Network inventory timeout') . "</td>";
             echo "<td>";
+            $general = __('General setup');
+            if (isset($CFG_GLPI['timeout_networkinventory'])) {
+                $general = sprintf('%1$s (%2$s)', $general, $CFG_GLPI['timeout_networkinventory']);
+            }
             Dropdown::showNumber(
                 'timeout_networkinventory',
                 [
-                'value' => $this->fields['timeout_networkinventory'],
-                'toadd' => [__('General setup')]
+                    'value' => $this->fields['timeout_networkinventory'],
+                    'toadd' => [-1 => $general]
```

Comment:
General should be bind to 0. and min set to 1.
```suggestion
                    'toadd' => [0 => $general],
                    'min' => 1
```